src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java

```diff
@@ -35,8 +34,7 @@ public static void main(String[] args) {
      * @param depositAnnualPercentage ежегодный простой процент
      */
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
+        return depositAmount + ((depositAmount * (float)depositAnnualPercentage / 100) * depositYears);
```

Comment:
"Code smell: Magic Number
Solution: Named Constants"

Reply:
this fixed